zerver/tests/test_email_notifications.py

```diff
@@ -186,7 +186,7 @@ def _test_cases(self, tokens: List[str], msg_id: int, body: str, email_subject:
         if settings.EMAIL_GATEWAY_PATTERN != "":
             reply_to_addresses = [settings.EMAIL_GATEWAY_PATTERN % (u'mm' + t,) for t in tokens]
             reply_to_emails = [formataddr(("Zulip", address)) for address in reply_to_addresses]
-        else:
+        else:  # nocoverage: TODO
```

Comment:
This nocoverage is in error; you shouldn't have removed the test for the case where we don't offer a direct reply feature.

I re-added the test (with a change to the text it checks for) and added this back.